.gitignore

```diff
@@ -9,3 +9,4 @@ docs/src
 *.swo
 *.swp
 /private/
+.idea/
```

Comment:
We should not add editor specific folders to gitignore. These things should be stored in your own global gitignore.

Reply:
Gotcha, seems like a good addition to contributers guide aswel!